### DISCUSSION TITLE
Deployment Diagramme

### DISCUSSION THREAD

**Discussion by raphaelauv** [fastapi/fastapi#7750] [Questions] [answered]
Hey, maybe a deployment diagramme could help everybody new in the back-end python community. (there is a lot of open source project and different lib/framework )

something like this ?

![fastAPI](https://user-images.githubusercontent.com/10202690/73546713-f86fed00-443d-11ea-834f-3cd22e607fcf.png)

**Comment by Kludex** [comment]
What do you think about this? @raphaelauv 
![image](https://user-images.githubusercontent.com/7353520/92994818-f1de4380-f4fd-11ea-8c12-cee3b973265e.png)

I found it here: https://towardsdatascience.com/build-and-host-fast-data-science-applications-using-fastapi-823be8a1d6a0

EDIT: I didn't read it correctly. The image I have is not the one you are looking for... Still cool anyway.

**Comment by raphaelauv** [comment, marked as answer]
More than saying what is what ( which is easy to find by reading the first line of respective readme ) , 

I was thinking about explaining :

- the flow of an http-request through all the components 
![Screenshot from 2020-09-12 14-08-28](https://user-images.githubusercontent.com/10202690/92995251-3b309200-f502-11ea-91a1-e33bfde864c8.png)

and
- in which component are wrapped (or run) each of them

**Comment by tiangolo** [comment]
It would make sense! I've added some Draw.io diagrams, I don't remember if I did that already in FastAPI but certainly did in SQLModel. This could go in the Deployment section. The Draw.io diagram file would need to be committed (not just the generated SVG) to be able to edit it later if necessary. If you would like to make a PR with that it would be great, but I imagine it's too late as this was a long time ago. 😅 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
